getgather/mcp/brand/goodreads.py

```diff
@@ -15,7 +16,12 @@
 @goodreads_mcp.tool(tags={"private"})
 async def get_book_list() -> dict[str, Any]:
     """Get the book list from a user's Goodreads account."""
-    return await extract()
+
+    path = os.path.join(os.path.dirname(os.path.dirname(__file__)), "patterns", "**/*.html")
+    patterns = load_distillation_patterns(path)
+
+    books = await run_distillation_loop("https://www.goodreads.com/review/list", patterns)
```

Comment:
Obtain the `browser_profile` first. If it's empty or default, throw an exception since for now the distillation-based tool can only reuse a "warm" browser profile.

Reply:
updated it on [006e3f6](https://github.com/mcp-getgather/mcp-getgather/pull/232/commits/006e3f626b8ccbd47d7a3aa1bea2c35b1bde3cc1)